pkg/autoscaler/collector.go

```diff
@@ -35,7 +35,7 @@ const (
 	scrapeTickInterval = time.Second
 
 	// BucketSize is the size of the buckets of stats we create.
-	BucketSize = 2 * time.Second
+	BucketSize = 1 * time.Second
```

Comment:
maybe set it equal to the `scrapeTickInterval` now?

Reply:
Sure makes sense,